packages/list-view/tests/list_view_test.js

```diff
@@ -56,13 +58,15 @@ test("should render an empty view when there is no content", function() {
         style: 'height:' + emptyViewHeight + 'px;'
       });
 
-  view = Ember.ListView.create({
-    content: content,
-    height: height,
-    rowHeight: rowHeight,
-    itemViewClass: itemViewClass,
-    emptyView: emptyView
-  });
+  Ember.run(function(){
+    view = Ember.ListView.create({
```

Comment:
its not a bug in your code, but it is making several issues in list-view apparent. I am also surprised this wasn't an issue pre the latest beta.

I need to grab dinner and stuff, I'll try to look more closely afterwords 

TL;DR initial render should create views with the right context, rather then create them, then add the context.